packages/io/x3d-deserializer/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+- allow commas as delimiter in MF fields
```

Comment:
Actually, the change logs are automatically updated when new releases are published to NPM. So, this change is not necessary.

there's some information here...

https://github.com/jscad/OpenJSCAD.org/wiki/Admin-:--Merging-Pull-Requests

Reply:
Ok. Thanks for the info.